metadatafields.md

```diff
@@ -71,29 +71,47 @@ Return codes:
 
 To create a metadata field, perform a post with the JSON below when logged in as admin.
 
-```
+```json
 {
   "element": "contributor",
   "qualifier": "tester",
-  "scopeNote": "An agent which provided illustrations for the resource",
-  "type": "metadatafield"
+  "scopeNote": "An agent which provided illustrations for the resource"
 }
 ```
 
+Return codes:
+* **201 Created** - if the operation succeed
+* **400 Bad Request** - if the body of the request can't be parsed (for example because of a missing field)
+* **401 Unauthorized** - if you are not authenticated
+* **403 Forbidden** - if you are not logged in with sufficient permissions
+* **415 Unsupported Media Type** - if the schema ID does not exist
```

Comment:
this is not appropriate as the 415 is related to the format of the payload 

> the server refuses to accept the request because the payload format is in an unsupported format

not the semantic value, this scenario must go in the 422